boards/nxp/rd_rw612_bga/dts/goworld_16880_lcm.overlay

```diff
@@ -17,6 +17,8 @@
 &lcdic {
 	status = "okay";
 	nxp,swap-bytes;
+	/* Raise the timer0 ratio to enable longer reset delay */
+	nxp,timer0-ratio = <15>;
```

Comment:
LGTM but what dictates using a value of 15?

Reply:
Yes- also this display driver uses a long reset pulse (100 ms) which requires us to increase the timer0 base value. The reset delay is given in units of the "Timer 0 base", the period of which is calculated as 2^(timer0-ratio)/lcdic_freq. So raising this value lets us generate longer reset pulses.